### PR TITLE
fix: allow unit tests to run

### PR DESCRIPTION
**Pull request by padamstx**
## PR summary
This PR modifies the data-virtualization module's pom.xml to allow unit tests to be executed.

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [ ] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type  

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] New tests
- [ ] Build/CI related changes
- [ ] Documentation content changes
- [ ] Other (please describe)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [ ] No

### CODE REVIEW

File: modules/data-virtualization/pom.xml [hunk 1]
```diff
@@ -63,13 +63,6 @@
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-surefire-plugin</artifactId>
 				<version>3.0.0-M1</version>
-				<dependencies>
```
Comment (padamstx) [inline modules/data-virtualization/pom.xml:66]: The SDK generator generates unit tests that use TestNG, not JUnit, so we need to just remove the junit reference here.

**Comment by padamstx** [comment]
@baijiebj please review and then merge... this should cause a new version of the project (0.0.7) to be created.

**Comment by baijiebj** [comment]
Thanks. I merged, and will check the result.

**Pull request merged**